packages/docs/README.md

```diff
@@ -1,26 +1,60 @@
-# ZeppelinOS documentation site
+# ZeppelinOS documentation website
 
-Centralized documentation site for ZeppelinOS.
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg)](https://github.com/RichardLitt/standard-readme)
 
-To view this website locally, clone the repository with:
+> Documentation for the ZeppelinOS smart contract platform.
 
-`git clone https://github.com/zeppelinos/zos-docs.git`
+## Development setup
+
+First, install [Node.js](http://nodejs.org/), [npm](https://npmjs.com/) and
+[git](https://git-scm.com/).
+
+Then, clone the repository:
+
+```sh
+git clone https://github.com/zeppelinos/zos-docs.git
+```
 
 Install Docusaurus:
 
-`npm install -g docusaurus-init`
+```sh
+npm install --global docusaurus-init
+```
 
 Navigate to the `docs/website` directory and run:
 
-`npm install`
+```sh
```

Comment:
we can add `cd /docs/docs/website` here to be explicit

Reply:
ah, yes, good idea.